node/src/server_initializer.rs

```diff
@@ -47,9 +47,11 @@ impl ServerInitializer for ServerInitializerReal {
                     .as_mut()
                     .initialize_as_privileged(&params.multi_config),
             );
-
+        //TODO check if we can remove chown function from code base
```

Comment:
I don't think we can. At the very least, we'll need to start writing to the logfile as `root`, or else we won't get any of our early operations logged. Just before we drop privileges, we'll have to `chown` the logfile to normal-user ownership.

Unless we can find a way around this, I think we have to keep `chown`.

I remember that at one point we tried to create a normal-user-owned file while we were `root`, and we ran into problems. I don't remember what problems; maybe they're worth exploring again, maybe not.